src/_sass/home2.scss

```
@@ -244,6 +244,8 @@
 // Features --------------------------------------------------
 .Home2-features {
   margin-bottom: $spacing-unit;
+  // TODO(dmiller): there's probably a better way to do this
+  height: 300px;
```

To make it roughly closer to the height of the menu...

```suggestion
  height: 400px;
```